frontend/src/views/analysis_view.js

```diff
@@ -219,6 +246,11 @@ export class AnalysisView extends React.Component {
                         value = {compute_median_view_time}
                         unit = {"seconds"}
                     />
+                    <SingleValueAnalysis
```

Comment:
nitpick -- no spaces around the = when assigning for props (see other examples here)...